examples/context_manager.py

```diff
@@ -0,0 +1,23 @@
+# Copyright Notice:
+# Copyright 2016-2019 DMTF. All rights reserved.
+# License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/python-redfish-library/blob/master/LICENSE.md
+
+import sys
+import redfish
+
+# When running remotely connect using the address, account name,
+# and password to send https requests
+login_host = "https://192.168.1.100"
+login_account = "admin"
+login_password = "password"
+
+## Create a REDFISH object
+with redfish.redfish_client(base_url=login_host, username=login_account, password=login_password) as REDFISH_OBJ:
+    # Login into the server and create a session
+    REDFISH_OBJ.login(auth="session")

```

Comment:
log is already done on __enter__ so you can skip it

Reply:
True... Hmmm, unfortunate that it defaults to Basic Auth for that... I can see people using sessions more....

Reply:
Maybe better to provide default auth method in c-tor `__init__` 
And login() will use it if not provided... 

Anyhow I'm glad with new example! 👍

Reply:
@Yergidy , do you have any opinion in changing the default authentication scheme to be session based? See https://github.com/DMTF/python-redfish-library/blob/master/src/redfish/rest/v1.py#L877